hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -25,5 +24,852 @@
       stage: V1
       tests: ["usbdev_smoke"]
     }
+    {
+      name: rst_ni
+      desc: '''
+            Verify the reset values of usbdev CSRs as indicated in the RAL specification.
+            
+            See hw/dv/tools/dvsim/testplans/csr_testplan.hjson for more description, This follows
+            the same approach, but applies to the USB regisers.
+            '''
+      stage: V1
+      tests: ["usbdev_rst_ni"]
+    }
+    {
+      name: intr_test 
+      desc: '''
+            Verify write only access of INTR_TEST register.
+            
+            - write to the INTR_TEST register.
+            - Verify that the writing value to the INTR_TEST register will force the
+              corresponding pins in the interrupt interface to HIGH.
+            '''
+      stage: V1
+      tests: ["usbdev_intr_test"]
+    }
+    {
+      name: alert_test 
```

Comment:
This can be covered by 
including "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson" to [testplan]()